tests/e2e/main.go

```diff
@@ -246,6 +248,12 @@ var stepChoices = map[string]StepChoice{
 		description: "test minting without inactive validators as a sanity check",
 		testConfig:  MintTestCfg,
 	},
+	"permissionless-ics": {
+		name:        "permissionless-ics",
+		steps:       stepsPermissionlessICS(),
```

Comment:
`stepsPermissionlessICS` does not seem to exist. Can you comment out with a `TODO (PERMISSIONLESS)`?